sound/soc/sof/sof-priv.h

```diff
@@ -415,6 +415,8 @@ struct snd_sof_dev {
 	u32 host_offset;
 	u32 dtrace_is_enabled;
 	u32 dtrace_error;
+	u32 dtrace_eof:1;		/* trace EOF pending */
```

Comment:
the bitfield is useless here, you'll still allocate 32 bits :-)

Reply:
@plbossart That is true. Ok, I'll modify. :)